src/app/(frontend)/_components/NavButton.tsx

```diff
@@ -0,0 +1,10 @@
+import Link from 'next/link'
+
+type NavButtonProps = {
+  children: React.ReactNode
+  route: string
+}
+
+export default function NavButton({ children, route }: NavButtonProps) {
```

Comment:
prefer named exports for components.
```suggestion
export function NavButton({ children, route }: NavButtonProps) {
```
also applies to other components